sdk/cosmosdb/cosmos/src/ClientContext.ts

```diff
@@ -46,6 +52,18 @@ export class ClientContext {
     this.connectionPolicy = cosmosClientOptions.connectionPolicy;
     this.sessionContainer = new SessionContainer();
     this.partitionKeyDefinitionCache = {};
+    this.pipeline = null;
```

Comment:
Ah, I see you're using `strict: false` in your tsconfig... I'd recommend you change that at some point in the future ;)